### PR TITLE
Minor Cleanup

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

File: db/migrate/20200425194536_remove_unused_profile_flags.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class RemoveUnusedProfileFlags < ActiveRecord::Migration[5.2]
+  def change
```
Comment (nilsding) [inline db/migrate/20200425194536_remove_unused_profile_flags.rb:2]: This change will break:
- data export
- the "Your data" view in the settings page
- everything else that makes use of them

**Review by nilsding — CHANGES_REQUESTED** [review]

File: db/schema.rb [hunk 2]
```diff
@@ -10,11 +10,32 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_04_19_185535) do
+ActiveRecord::Schema.define(version: 2020_04_25_194536) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "active_storage_attachments", force: :cascade do |t|
```
Comment (nilsding) [inline db/schema.rb]: Where did this come from?

**Review by nilsding — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 2 above)
Comment (raccube) [reply db/schema.rb]: AAAA

Comment (raccube) [reply db/schema.rb]: Need to nuke my local DB

File: app/views/user/_modal_privileges.html.haml [hunk 3]
```diff
@@ -8,11 +8,7 @@
         %h4#modal-privileges-label.modal-title
           = raw t('views.actions.privilege', user: @user.screen_name)
       %ul.list-group.groups--list
-        = render 'user/modal_privileges_item', privilege: 'blogger', description: t('views.modal.privilege.blogger'), user: @user
```
Comment (nilsding) [inline app/views/user/_modal_privileges.html.haml:11]: Please remove the code that assigns these

**Review by nilsding — CHANGES_REQUESTED** [review]

File: app/views/user/_modal_privileges.html.haml (hunk 3 above)
Comment (mltnhm) [reply app/views/user/_modal_privileges.html.haml:11]: Refresh :)

File: Rakefile [hunk 4]
```diff
@@ -501,7 +441,7 @@ namespace :justask do
     export_dirname = "export_#{u.screen_name}_#{Time.now.to_i}"
     export_filename = u.screen_name
 
-    %i(id screen_name display_name created_at sign_in_count last_sign_in_at friend_count follower_count asked_count answered_count commented_count smiled_count motivation_header bio website location moderator admin supporter banned blogger).each do |f|
+    %i(id screen_name display_name created_at sign_in_count last_sign_in_at friend_count follower_count asked_count answered_count commented_count smiled_count motivation_header bio website location moderator admin banned).each do |f|
```
Comment (nilsding) [inline Rakefile]: just remove this whole task -- it's been made redundant by the sidekiq-based export job anyway

File: app/controllers/ajax/moderation_controller.rb [hunk 5]
```diff
@@ -164,9 +164,9 @@ def privilege
     target_user = User.find_by_screen_name(params[:user])
 
     @message = I18n.t('messages.moderation.privilege.nope')
-    return unless %w(blogger supporter moderator admin contributor translator).include? params[:type].downcase
+    return unless %w(moderator admin).include? params[:type].downcase
 
-    if %w(supporter moderator admin).include?(params[:type].downcase) && !current_user.has_role?(:administrator)
+    if %w(moderator admin).include?(params[:type].downcase) && !current_user.has_role?(:administrator)
```
Comment (nilsding) [inline app/controllers/ajax/moderation_controller.rb]: the first part of this `if` can never be false (already checked by the `return unless ...` above) --> remove it

File: app/controllers/ajax/moderation_controller.rb [hunk 6]
```diff
@@ -176,16 +176,8 @@ def privilege
...
-      target_user.blogger = status
-    when 'contributor'
-      target_user.contributor = status
-    when 'translator'
-      target_user.translator = status
-    when 'supporter'
-      target_user.supporter = status
-    when 'moderator', 'admin'
+
+    if %w(moderator admin).include? type
```
Comment (nilsding) [inline app/controllers/ajax/moderation_controller.rb]: again, this `if` is redundant

File: db/migrate/20200425194536_remove_unused_profile_flags.rb [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+    remove_column :users, :admin
```
Comment (nilsding) [inline db/migrate/20200425194536_remove_unused_profile_flags.rb:3]: still referenced in `app/controllers/static_controller.rb`

File: db/migrate/20200425194536_remove_unused_profile_flags.rb [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+    remove_column :users, :blogger
+    remove_column :users, :contributor
```
Comment (nilsding) [inline db/migrate/20200425194536_remove_unused_profile_flags.rb:5]: still referenced in `app/views/static/about.html.haml`

**Review by nilsding — CHANGES_REQUESTED** [review]

**Review by nilsding — APPROVED** [review]

**Pull request merged**